docs/video/obs/video.md

```diff
@@ -4,16 +4,20 @@ icon: obs/video
 
 # Difference between Base (Canvas) and Output (Scaled) resolutions
 
-As canvas implies, it's the resolution of the preview pane where you place sources
+As canvas implies, it's the resolution of the preview pane where you place sources/
 
-The output takes the base canvas and scales it to what 
+The output takes the base canvas and scales it to whatever you set it to.
 
 ### When using a different resolution is worth using
 
-If you're limited on bandwith / encoding efficiency you might consider scaling down, or the opposite on YouTube for extra bitrate (same principle as (up)scaling)
+If you're limited on bandwidth / encoding efficiency you might consider scaling down, or the opposite on YouTube for extra bitrate (same principle as (up)scaling)
 
-For recording this won't grant you
+For recording it's not recommended to scale at all.
+
+<!--
+Maybe in some situations this isn't true, I've never personally had to.
```

Comment:
my bad i didn't finish this one, yeah it can be useful e.g if you're working on a 720p project and must respect that resolution when recording and don't wanna bother doing it later